src/dehtml.rs

```diff
@@ -2,18 +2,49 @@
 //!
 //! A module to remove HTML tags from the email text
 
+use std::io::BufRead;
+
 use once_cell::sync::Lazy;
-use quick_xml::events::{BytesEnd, BytesStart, BytesText};
+use quick_xml::{
+    events::{BytesEnd, BytesStart, BytesText},
+    Reader,
+};
 
 static LINE_RE: Lazy<regex::Regex> = Lazy::new(|| regex::Regex::new(r"(\r?\n)+").unwrap());
 
 struct Dehtml {
     strbuilder: String,
     add_text: AddText,
     last_href: Option<String>,
+    // Some providers wrap a quote in <div name="quote">. After a <div name="quote">, this count is
```

Comment:
Some providers = gmx? If so, I think it makes sense to add this to the comments, so we don't wonder why this code is still around when they change the layout of their HTML.